### DISCUSSION TITLE
Alternating background pattern / multiple <Background/> components possible?

### DISCUSSION THREAD

**Discussion by elringus** [xyflow/xyflow#2937] [Q&A] [answered]
I'm trying to achieve following background pattern:

![img](https://user-images.githubusercontent.com/2056864/226382456-0ec36ab6-70d9-4c6f-97e7-396658f33e80.png)

Was hoping I could use 2 line `<Background/>`  with different gaps, but it doesn't seem to be supported (only the first one is rendered). Is there a built-in way to achieve the alternating pattern?

**Comment by moklick** [comment, marked as answer]
we don't have it. I think the easiest option would be to copy the current background component. I am also open for a PR for this one if it's possible to add as another variant.
